lib/Remote/RemoteTargetLocator.php

```diff
@@ -63,10 +68,20 @@ public function defaultContent()
      */
     public function frame($frame)
     {
-        if ($frame instanceof WebDriverElement) {
-            $id = ['ELEMENT' => $frame->getID()];
+        if ($this->isW3cCompliant) {
+            if ($frame instanceof WebDriverElement) {
+                $id = [JsonWireCompat::WEB_DRIVER_ELEMENT_IDENTIFIER => $frame->getID()];
+            } else if (null === $frame) {
```

Comment:
```suggestion
            } elseif ($frame === null) {
```

Please don't use yoda conditions, they are harder to read :).